logstash-core/lib/logstash/modules/importer.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/logging"
+
+module LogStash module Modules class Importer
+  include LogStash::Util::Loggable
+
+  attr_reader :client
```

Comment:
I believe so, yes.  @guyboertje wrote this, and the client is used outside, if memory serves.